src/main/java/com/cs56fitnessapp/models/workout/StrengthTraining.java

```diff
@@ -0,0 +1,90 @@
+package com.cs56fitnessapp.models.workout;
+
+import com.cs56fitnessapp.models.User;
+import java.time.LocalDate;
+
+/**
+ * @author Daniel Cervantes
+ * @author Ksenia Koldaeva
+ * Date created: 10/18/17
+ * Last updated: 11/11/17
+ */
+
+public class StrengthTraining extends Workout {
+
+    private StrengthTrainingLevel strengthTrainingLevel;
+    private boolean training;
+
+    /**
+     * Constructs Strength Training object with provided parameters
+     * @param user user performing Strength Training
+     * @param date date of Strength Training
+     * @param timePerformingHours amount of time of Strength Training in hours
+     * @param strengthTrainingLevel one of three different kinds of strength training levels
+     */
+
+    public StrengthTraining(User user, LocalDate date, double timePerformingHours, StrengthTrainingLevel strengthTrainingLevel) {
+        super(user, date);
+        this.setTimePerformingHours(timePerformingHours);
+        this.strengthTrainingLevel = strengthTrainingLevel;
+
+    }
+
+    // Setters and Getters
+    public StrengthTrainingLevel getStrengthTrainingLevel() {
+        return strengthTrainingLevel;
+    }
+
+    public void setStrengthTrainingLevel(StrengthTrainingLevel strengthTrainingLevel) {
+        this.strengthTrainingLevel = strengthTrainingLevel;
+    }
+
+    public boolean isTraining() {
+        return training;
+    }
+
+    public void setTraining(boolean training) {
+        this.training = training;
+    }
+
+
+    /**
+     * @return calories out from Strength Training based on
+     * the level of weights that were used during the workout
+     */
+
+    @Override
+    public int getCaloriesOut() {
+        int caloriesOut = 0;
+
+        switch (strengthTrainingLevel) {
+            case HEAVY:
+                if (training) {
+                    caloriesOut = FitnessFormulas.caloriesOutByMET(FitnessFormulas.MET_STRENGTH_TRAINING_HEAVY, this.getUser().getbodyMassKg(), this.getTimePerformingHours());
+                }
+                break;
```

Comment:
Here `if (training) ` introduces confusion. With the timer implementation, I imagine we must have a thread with a while loop to change calories out put while user is training. 
In this implementation the logic is different. 
I think `if (training)`  should be removed.